server/views/pages/add-steps.njk

```diff
@@ -127,11 +132,9 @@
                             }) }}
                         </dt>
                         <dt class="govuk-summary-list__value">
-                            {% if not loop.first %}
-                                <button value="remove-step-{{ loop.index }}" type="submit" name="action" class="govuk-link govuk-!-margin-bottom-0" data-module="govuk-button">
-                                    {{ locale.removeStepLinkButton }}
-                                </button>
-                            {% endif %}
+                            <button value="remove-step-{{ loop.index }}" type="submit" name="action" class="govuk-link govuk-!-margin-bottom-0" data-module="govuk-button">
```

Comment:
The Figma says we shouldn't display Remove Step for the first step e.g. https://www.figma.com/design/Ay0IR3IbHGQvznEWRSDUro/Update-a-plan?node-id=1463-15150&t=kOoaTc7OKDKZAGV0-4

Reply:
This is now open as https://dsdmoj.atlassian.net/browse/SP2-748